composer.json

```diff
@@ -25,7 +27,8 @@
         "phpunit/phpunit": "^6.5",
         "se/selenium-server-standalone": "^2.52",
         "sylius-labs/coding-standard": "^1.0",
-        "symplify/easy-coding-standard": "^2.4"
+        "symplify/easy-coding-standard": "^2.4",
```

Comment:
There is no need to require 'easy-coding-standard' because 'solutiondrive/coding-standard' requires the package by itself and perhaps this will conflict in future versions